repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tasks/ClassificationTask.java

```diff
@@ -100,7 +104,7 @@ public AtlasTask.Status perform() throws AtlasBaseException {
 
         try {
             setStatus(IN_PROGRESS);
-
+            setAssetsCountToPropagate(1234L);
```

Comment:
Let's remove this code which was used for debugging.

Reply:
Removed, thanks for pointing out